.azuredevops/pipelineTemplates/jobs.dependenciesModuleDeployment.yml

```
@@ -121,6 +121,13 @@ jobs:
                   $targetFileList += '${{ deploymentBlock.path }}'
                 }
 
+                # Collection additional information
+                ## Service Principal Object ID
```

Here and in other occurrences
```suggestion
                # Get Service Principal Object ID
```

---

I'd also suggest to always place this snippet either before or after the `# Get target files` one

---

Addressed